Source/DafnyCore/Compilers/ResolvedDesugaredExecutableDafny/Compiler-fdafny.cs

```diff
@@ -0,0 +1,20 @@
+using System;
+using System.IO;
+using System.Linq;
+using Dafny;
+using FDafnyPlugin;
+
+namespace Microsoft.Dafny.Compilers;
+
+class RDEDafnyCompiler : DafnyWrittenCompiler {
```

Comment:
Please either use full words or add a comment about what the abbreviation means.